xdsl/transforms/__init__.py

```diff
@@ -558,5 +563,6 @@ def get_varith_fuse_repeated_operands():
         "test-add-timers-to-top-level-funcs": get_test_add_timers_to_top_level_funcs,
         "test-lower-linalg-to-snitch": get_test_lower_linalg_to_snitch,
         "eqsat-create-eclasses": get_eqsat_create_eclasses,
+        "eqsat-add-costs": get_eqsat_add_costs,
```

Comment:
I thought this list was meant to roughly be in alphabetical order but it seems not looking at the rest of it

Reply:
Yeah it got messed up recently, I want to do a separate PR sorting this list, I'll open it in parallel to this and adjust the one that gets approved second.

Reply:
#3426